spec/neo4j-core/unit/query_spec.rb

```diff
@@ -639,6 +639,14 @@ def self.it_generates(cypher, params = {})
     describe '.create(q: {Person: {age: nil, height: 70}})' do
       it_generates 'CREATE (q:`Person` {age: {q_Person_age}, height: {q_Person_height}})', q_Person_age: nil, q_Person_height: 70
     end
+
+    describe ".create(q: {:'Child:Person' => {age: 41, height: 70}})" do
+      it_generates 'CREATE (q:`Child`:`Person` {age: {q_Child_Person_age}, height: {q_Child_Person_height}})', q_Child_Person_age: 41, q_Child_Person_height: 70
+    end
+
+    describe ".create(:'Child:Person' => {age: 41, height: 70})" do
+      it_generates 'CREATE (:`Child`:`Person` {age: {Child_Person_age}, height: {Child_Person_height}})', Child_Person_age: 41, Child_Person_height: 70
+    end
```

Comment:
OK, I think what I did supports your idea of an array of sym's.  The %i version didn't work out of the box and it seems like the array of sym's is an improvement at least and I didn't want to dig further without a clear need - hope that's ok :-)